app/client/components/cancel/cancellationSummary.tsx

```diff
@@ -11,22 +15,39 @@ import { GenericErrorScreen } from "../genericErrorScreen";
 import { PageContainer } from "../page";
 import { ResubscribeThrasher } from "../resubscribeThrasher";
 import { SupportTheGuardianButton } from "../supportTheGuardianButton";
+import { hrefStyle } from "./cancellationConstants";
 import { CancellationReasonContext } from "./cancellationContexts";
 
 const actuallyCancelled = (
   productType: ProductType,
   productDetail: ProductDetail
 ) => {
   const deliveryRecordsLink: string = `/delivery/${productType.urlPart}/records`;
+  const subscription = productDetail.subscription;
   return (
     <>
       <PageContainer>
         <h3>Your {productType.friendlyName} is cancelled.</h3>
         {productType.cancellation && (
           <p>
-            {productType.cancellation.summaryMainPara(
-              productDetail.subscription
-            )}
+            {productType.cancellation?.alternateSummaryMainPara ||
+              (subscription.end ? (
+                <>
+                  You will continue to receive the benefits of your{" "}
+                  {productType.friendlyName} until{" "}
+                  <b>{formatDate(subscription.end)}</b>. You will not be charged
+                  again. If you think you’re owed a refund, please contact us at{" "}
+                  <a
+                    css={hrefStyle}
+                    href="mailto:customer.help@theguardian.com"
+                  >
+                    customer.help@theguardian.com
+                  </a>
+                  .
+                </>
+              ) : (
+                "Your cancellation is effective immediately."
+              ))}
```

Comment:
made this generic and moved it here from it being repeated in `productTypes.tsx`